src/proctored-exam-settings/ProctoredExamSettings.jsx

```diff
@@ -18,8 +18,6 @@ function ExamSettings(props) {
   const [allowOptingOut, setAllowOptingOut] = useState(false);
   const [proctoringProvider, setProctoringProvider] = useState('');
   const [availableProctoringProviders, setAvailableProctoringProviders] = useState([]);
-  // TODO: we'll probably want to hide this field when proctortrack is not selected; currently,
-  // this causes some errors in the browser console
```

Comment:
is this a ticket, or a sub-task for an existing ticket? Mostly commenting on this comment since now's probably a good time in the process to figure out a place in JIRA for this thought to live.

Reply:
There was a ticket for this work, which has been completed ([MST-335](https://openedx.atlassian.net/browse/MST-335)), but this comment wasn't removed as part of the pull request, so I was just cleaning it up. Moving forward, we'll include ticket numbers in the comment to more clearly tie the code to tickets.